src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -138,7 +142,7 @@ export class StudentProfilePageComponent implements OnInit {
           }, (response: ErrorMessageOutput) => {
             this.statusMessageService.showErrorMessage(response.error.message);
           });
-    });
+    }).catch(() => {});
```

Comment:
I think it is better to supply empty handler for `.then(onFullfill, onReject)` rather than use `.catch()`. `catch` is used for error-handling. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

Reply:
@crizyli Reminder :P